src/modify_resources.py

```diff
@@ -66,14 +70,15 @@ def reset_egress_network_bandwidth(ssh_client, container_id):
     _,_,err_val_rate = ssh_client.exec_command(reset_network_cmd)
     if len(err_val_rate.readlines()) != 0:
         print 'ERROR: Resetting of container id {} network failed'.format(container_id)
+        print 'ERROR MESSAGE: {}'.format(err_val_rate)
```

Comment:
Not actionable in this PR but we need some way to indicate which resource stressing doesn't work for whatever reason.